lk2nd/device/dts/msm8916/msm8916-samsung.dts

```diff
@@ -841,4 +841,23 @@
 			};
 		};
 	};
+	
+	grandmaxltezc {
+		model = "Samsung Galaxy Grand Max (SM-G7200)";
+		compatible = "samsung,grandmaxltechn", "samsung,grandmax";
+		lk2nd,match-bootloader = "G7200*";
+
+		lk2nd,dtb-files = "msm8916-samsung-grandmax";
+
+		qcom,msm-id = <QCOM_ID_MSM8916 0>;
+		qcom,board-id = <0xCE08FF01 8>;
```

Comment:
```suggestion
		qcom,board-id = <0xCE08FF01 4>;
```

The revision is probably not correct, so `fastboot boot boot.img` would not work.
Grand Max is known to be revision `4`.

Ref: https://lxr.trvn.ru/linux-downstream/SM-G720AX/source/arch/arm/boot/dts/samsung/msm8916/msm8916-sec-grandmax-r04.dts#L24